_opensearch/rest-api/_script-apis/get-script-language.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: default
+title: Get Script Language
+parent: Script APIs
+grand_parent: REST API reference
+nav_order: 6
+---
+
+## Get script language
+
+Retrieves all supported script languages and their contexts.
+
+#### Sample request
+
+```json
+GET _script_language
+```
+
+#### Sample response
+
+The `GET _script_language` request returns the available contexts for each language:
+
+```json
+{
+  "types_allowed" : [
+    "inline",
+    "stored"
+  ],
+  "language_contexts" : [
+    {
+      "language" : "expression",
+      "contexts" : [
+        "aggregation_selector",
+        "aggs",
+        "bucket_aggregation",
+        "field",
+        "filter",
+        "number_sort",
+        "score",
+        "terms_set"
+      ]
+    },
+    {
+      "language" : "mustache",
+      "contexts" : [
+        "template"
+      ]
+    },
+    {
+      "language" : "opensearch_query_expression",
+      "contexts" : [
+        "aggs",
+        "filter"
+      ]
+    },
+    {
+      "language" : "painless",
+      "contexts" : [
+        "aggregation_selector",
+        "aggs",
+        "aggs_combine",
+        "aggs_init",
+        "aggs_map",
+        "aggs_reduce",
+        "analysis",
+        "bucket_aggregation",
+        "field",
+        "filter",
+        "ingest",
+        "interval",
+        "moving-function",
+        "number_sort",
+        "painless_test",
+        "processor_conditional",
+        "score",
+        "script_heuristic",
+        "similarity",
+        "similarity_weight",
+        "string_sort",
+        "template",
+        "terms_set",
+        "trigger",
+        "update"
+      ]
+    }
+  ]
+}
+```
+
+### Response fields
+
+The request contains the following response fields:
```

Comment:
Use a period instead of a colon before tables

Reply:
Done - thank you

Reply:
actually, it should be a colon per our style guide: https://github.com/opensearch-project/documentation-website/blob/main/STYLE_API_TEMPLATE.md

**Sample response**
Include a JSON example response to show what the API returns. See the examples below.

"The GET /sample-index1/_settings request returns the following response fields:"

Reply:
@ariamarble @kolchfa-aws @alicejw-aws If there’s text that introduces a table, it should be a complete sentence and end with a period, not a colon.